src/include/index/bwtree.h

```diff
@@ -2864,6 +2866,21 @@ class BwTree : public BwTreeBase {
    * the mapping table rather than CAS with nullptr
    */
   void InitMappingTable() {
+    mapping_table = (std::atomic<const BaseNode *> *) \
+                    mmap(NULL, 1024 * 1024 * 1024, 
```

Comment:
1. The value (1024*1024*1024) shouldn't be hardcoded (the MAPPING_TABLE_SIZE is #defined to 1MB). 

2. InitMappingTable() seems to be invoked from the bwtree constructor - is 1GB mmaped for every index created?